public/pages/cadastro/cadastro_pais.html

```diff
@@ -181,7 +181,7 @@ <h5  class="title text-secondary">
     <hr>
     <section class ="d-flex justify-content-center align-items-center">
       &copy: Squad 61.
-      <img width="60px + 15vw" src="/public/assets/images/logosquad.png">
+      <img width="60px + 15vw" src="./public/assets/images/logosquad.png">
```

Comment:
Sugestão de correção:

```suggestion
      <img class="img-fluid" src="../../../assets/images/logosquad.png">
```